Firestore/Swift/Source/Codable/DocumentID.swift

```diff
@@ -108,8 +108,8 @@ protocol DocumentIDProtocol {
 ///   its value is ignored. This allows you to read a document from one path and
 ///   write it into another without adjusting the value here.
 @propertyWrapper
-public struct DocumentID<Value: DocumentIDWrappable & Codable>:
-  StructureCodingUncodedUnkeyed {
+public struct DocumentID<Value: DocumentIDWrappable & Codable & Sendable>:
```

Comment:
I'm not sure it can be Sendable because the public `wrappedValue` API with a setter makes it mutable.